Plots/Panels/NCL_panel_11.py

```diff
@@ -0,0 +1,180 @@
+"""
+NCL_panel_11.py
+===============
+This script illustrates the following concepts:
+    - Specifying how many and where to draw plots using gridspec
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/panel_11.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/panel_11_1_lg.png and https://www.ncl.ucar.edu/Applications/Images/panel_11_2_lg.png
+"""
+
+###############################################################################
+# Import packages
+
+import numpy as np
+import xarray as xr
+import matplotlib.pyplot as plt
+
+import geocat.datafiles as gdf
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine
+# and load the data into xarrays
+ds = xr.open_dataset(gdf.get('netcdf_files/chi200_ud_smooth.nc'))
+
+lon = ds.lon
+times = ds.time
+scale = 1000000
+chi = ds.CHI
+chi = chi / scale
+
+###############################################################################
+# Plot:
+
+
+def Plot(ax_list):
```

Comment:
Generally, functions should be lowercase by convention. Also, the name "Plot" kind of vague, especially with the default matplotlib call also being called "plot".